README.md

```diff
@@ -130,8 +130,8 @@ following comprehensive example:
 
 ### Example App
 
-The following app showcases the usage of the `LovelyEditor` with two configured
-with pre-configured Editors (in this case `EditorQuill` and `EditorTui`) and
+The following app showcases the usage of the `LovelyEditor` with one configured
+with pre-configured Editor (in this case `EditorQuill`) and
```

Comment:
Rethink the whole sentence as it was not proper English even before your change.